content/en/checkout/reference.md

```diff
@@ -15,76 +15,101 @@ Creates a Checkout Session.
 
 #### Parameters:
 
-<!-- TODO: Required Vs Optional, it's unclear -->
-<!-- TODO: Better way to show this! -->
-
 An object with the following properties:
 
-- `type` (*required*): `Array` of `string`s representing which [payment type(s)](/concepts/payment-types) this Checkout session should accept. For now, only `['single']` is supported.
-- `payment` (*required*): `Object` describing the payment configuration, with the following properties:
-  - `methods`: `Array` of `string`s with the payment methods accepted in this Checkout session. The available ones are:
-    - `'cc'` (Credit card)
-    - `'mbw'` (MB WAY)
-    - `'mb'` (Multibanco)
-    - `'dd'` (Direct Debit)
-    - `'vi'` (Virtual IBAN)
-    - `'uf'` (Universo Flex)
-    - `'sc'` (Santander Consumer)
-  - `type`: `string` indicating the type of payment for Credit Card and MB WAY operations. Either `'sale'` (default) or `'authorisation'`.
-  - `capture`: `Object` required if the payment `type` is sale.
+<div class="ep-protocol-list">
+
+- `type`: `Array` with a `string` representing which [payment type](/concepts/payment-types) this Checkout session should accept. Either `['single']` or `['frequent']` are supported.
+  Only the first value in the `Array` is considered. (*required*)
```

Comment:
Add an example to the end of line 23. Like: `(Ex: ['single', 'frequent'])` just to create a clear image of this situation on the person who's reading.